qemu/tests/cfg/usb_device_check.cfg

```diff
@@ -85,3 +86,18 @@
         - input_devices_under_two_tier_hub:
             no usb-ehci
             usb_topology = '{"usb-hub":2,"usb-mouse":1,"usb-kbd":1,"usb-tablet":1}'
+        - max_devices_under_one_hub:
+            no usb-ehci
+            usb_topology = '{"usb-hub":1,"usb-mouse":8}'
+        - usb_mouse:
+            Host_RHEL.m6:
+                no usb-ehci
+            usb_topology = '{"usb-mouse":1}'
+        - usb_mouse_and_tablet:
+            Host_RHEL.m6:
+                no usb-ehci
+            usb_topology = '{"usb-mouse":1,"usb-tablet":1}'
+        - usb_kbd:
+            Host_RHEL.m6:
```

Comment:
@hereischen There is a bug on rhel6, would you also want to fix it in the patch?

Reply:
updated